### PR TITLE
fix: spelling errors in the PR template

### PR DESCRIPTION
**Pull request by Panquesito7**
## Changes proposed

- Fix spelling errors in the PR template.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/692"></a>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- If this pull requests solves an issue, as an example, use `Closes #31`. #31 stands for the issue number -->
+<!-- If this pull request solves an issue, as an example, use `Closes #31`. #31 stands for the issue number -->
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the Issue. #999 stands for the issue number you are fixing -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -8,15 +8,15 @@
 
 ## Changes proposed
 
-<!-- Here comes all the changes proposed through this PR -->
+<!-- Here come all the changes proposed through this PR -->
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
<!-- List all the changes proposed in your PR -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -8,15 +8,15 @@
...
 
 <!-- Check all the boxes which are applicable to check the box correct follow the following conventions -->
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -8,15 +8,15 @@
...
 <!--
 [x] - Correct
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:15]:
```suggestion
[x] - Marked as done
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -8,15 +8,15 @@
...
 [X] - Correct
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
[ ] - Marked as **not** done
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -8,15 +8,15 @@
...
 -->
 
-## Check List (Check all the boxes which are applicable) <!-- Follow the above conventions to check the box -->
+## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:19]:
```suggestion
## Check List (Mark all the applicable boxes) <!-- Follow the conventions mentioned above to mark the boxes -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -31,4 +31,4 @@
 
 ## Note to reviewers
 
-<!-- Add a note to reviewers if applicable -->
+<!-- Add notes to reviewers if applicable -->
```
Comment (krishguptadev) [inline .github/PULL_REQUEST_TEMPLATE.md:34]:
```suggestion
<!-- Add notes for the reviewers if needed -->
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -8,15 +8,17 @@
...
+[X] - Correct; marked as done
+
+[ ] - Correct; marked as **not** done
 -->
 
-## Check List (Check all the boxes which are applicable) <!-- Follow the above conventions to check the box -->
+## Check List (Check all the applicable boxes) <!-- Follow the above conventions to check the box -->
 
 - [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation.
```
Comment (starlightknown) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: maybe change/changes? (could be multiple too)

Comment (krishguptadev) [reply .github/PULL_REQUEST_TEMPLATE.md:24]:
```suggestion
- [ ] My change requires change(s) to the documentation.
```

**Review by krishguptadev — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by qwe123coder — APPROVED** [review]
LGTM :rocket:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -8,18 +8,20 @@
...
 
-<!-- Check all the boxes which are applicable to check the box correct follow the following conventions -->
+<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
 <!--
-[x] - Correct
-[X] - Correct
+[x] - Correct; marked as done
+[X] - Correct; marked as done
+
+[ ] - Correct; marked as **not** done
```
Comment (schmelto) [inline .github/PULL_REQUEST_TEMPLATE.md:18]:
```suggestion
[ ] - Not correct; marked as **not** done
```

**Review by thepranaygupta — APPROVED** [review]
LGTM 🚀

**Pull request merged**